### PR TITLE
Longer limit on task_id names

### PR DESCRIPTION
**Pull request by b11z**

### CODE REVIEW

File: fireflower/models.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ class TaskOutput(FireflowerDeclBase):
     __tablename__ = 'task_outputs'
 
     id = Column(Integer, primary_key=True)
-    task_id = Column(String(255), unique=True)
```
Comment (kkwteh) [inline fireflower/models.py]: One thing I learned recently is that there is no performance penalty for using the Text type over String. Can we just make this a Text?

Comment (kkwteh) [inline fireflower/models.py]: That is, there's no penalty in Postgres.

**Comment by kkwteh** [comment]
Lgtm

File: fireflower/models.py (hunk 1 above)
Comment (b11z) [inline fireflower/models.py]: :+1:

**Pull request merged**